css/custom.css

```diff
@@ -1,252 +1,98 @@
+@import url('../webfonts/Montserrat/Montserrat-Regular.ttf');
+
+/* change font for site */
 body {
-    padding-top: 0px;
+  font-family: 'Montserrat', sans-serif;
 }
 
+/* increase font size of paragraph content */
 body p {
   font-size:1.2em;
 }
 
-/*--------------------NAV LOGO--------------------------*/
-.navbar-brand{
-	min-height: 50px;
-    display: flex;
-    align-items: center;
+/* increase font-size of lists */
+main li {
+  font-size:1.2em;
+}
+
+/* add space between navbar and content */
+main {
+  padding-top: 2rem;
 }
 
-/* --------------------NAV BAR--------------------------*/
+/* make jumbotron slightly smaller */
+.jumbotron {
+    padding: 2rem;
+}
+
+/* change the navbar background color */
 .navbar-inverse {
   background-color: #57068c;
-  border-color: transparent;
-  font-size:1.3em;
-}
-.navbar-inverse .navbar-brand {
-  color: #ffffff;
 }
-.navbar-inverse .navbar-brand:hover,
-.navbar-inverse .navbar-brand:focus {
+
+/* change the link color and font in navbar*/
+.navbar-inverse .navbar-nav .nav-link {
   color: #ffffff;
-  background-color: transparent;
+  font-size:1.4em;
+  font-family: 'Montserrat', sans-serif;
 }
-.navbar-inverse .navbar-text {
-  color: #bbbbbb;
+
+/* change the color of active or hovered links in navbar */
+.navbar-inverse .nav-item.active .nav-link,
+.navbar-inverse .nav-item:focus .nav-link,
+.navbar-inverse .nav-item:hover .nav-link {
+    background-color: blue;
 }
-.navbar-inverse .navbar-nav > li > a {
-  color: #ffffff;
+
+/* change background of active links in navbar */
+.nav-item.active {
+	background-color: blue;
 }
-.navbar-inverse .navbar-nav > li > a:hover,
-.navbar-inverse .navbar-nav > li > a:focus {
-  color: #ffffff;
-  background-color: transparent;
+
+.nav.navbar-nav.ml-auto > .nav-item > .nav-link {
+  font-size:2em;
```

Comment:
This seems to be the difference between the two. It causes the social icons to have a different size, and the overall navbar too.